src/main/webapp/WEB-INF/static/js/trialmanager/subobservations/subObservationSet.js

```diff
@@ -1710,6 +1710,10 @@
 						$scope.observation.onOpenClose(isOpen);
 					}
 
+					$scope.methodChanged = function (item, model) {
+
```

Comment:
Is this supposed to be an empty method? If so put a comment why it's empty

Reply:
This is done, I just made the onMethodSelect in methods-select component optional, so we don't need to pass an empty function.